registry/storage/driver/s3-aws/s3.go

```diff
@@ -341,6 +342,23 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 
 	sessionToken := ""
 
+	virtualHostedStyleBool := false
+	virtualHostedStyle := parameters["virtualHostedStyle"]
```

Comment:
This needs to be all lowercase - there's logic elsewhere which translates environment variables into the parameters map. Doing so allows the environment variable to become `REGISTRY_STORAGE_S3_VIRTUALHOSTEDSTYLE`.

Reply:
updated in next push